src/routes/chatRoutes.js

```diff
@@ -6,7 +6,8 @@ router.get('/:albumId', async (req, res) => {
   const { albumId } = req.params;
   const messages = await ChatMessage.find({ albumId })
     .sort({ timestamp: -1 }) // Sort messages in reverse chronological order
```

Comment:
I'm not sure if you actually want these in reverse chronological order. When you'res showing them in the frontend, this ends meaning the most recent messages are scrolled all the way at the top of the chatbox

<img width="937" alt="Screenshot 2023-11-12 at 11 29 44 AM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/51d3347e-c57f-4cb4-b2ed-ad74b143619e">